ui/src/components/RadioConfirm/index.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import * as Stores from '@stores';
+
+type Props = {
+    status: boolean | undefined;
```

Comment:
This prop must be a boolean, always. Otherwise the radio input will switch from uncontrolled to controlled input and it will misbehave. There's a warning into the console about this.